.changeset/quick-pillows-jump.md

```diff
@@ -0,0 +1,15 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+"__section": tree
+---
+
+New allowUnused utility function
+
+The new `allowUnused` utility function discards its type or runtime argument.
```

Comment:
FYI: the PR description includes a version of this text with a typo ("it" instead of "its").